src/textEditor.ts

```diff
@@ -140,6 +141,32 @@ export class TextEditor {
     return vscode.window.activeTextEditor.document.getText(selection);
   }
 
+  /**
+   *  Retrieves the current word at position.
+   *  If current position is whitespace, selects the right-closest word
+   */
+  static getWord(position: Position) : string {
+    const whitespaceRegex = new RegExp("^ *$");
```

Comment:
I mean the Configuration object: https://github.com/VSCodeVim/Vim/blob/master/src/configuration/configuration.ts

And just this regex.